### PR TITLE
docs: improve internet jsdocs

### PR DESCRIPTION
**Pull request by ST-DDT**
Ref: #203

### CODE REVIEW

**Review by Shinigami92 — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

File: src/internet.ts [hunk 1]
```diff
@@ -191,12 +30,16 @@ export class Internet {
...
+   * @param lastName The optional last name to use. If not specified, a random one will be chosen.
+   * @param provider The mail provider domain to use. If not specified, a random free mail provider will be chosen.
    *
-   * @method faker.internet.email
-   * @param firstName
-   * @param lastName
-   * @param provider
+   * @example
+   * faker.internet.email() // 'Kassandra4@hotmail.com'
+   * faker.internet.email('Jeanne', 'Doe') // 'Jeanne63@yahoo.com'
```
Comment (prisis) [inline src/internet.ts:41]: Is it me or is Doe missing?

File: src/internet.ts [hunk 2]
```diff
@@ -214,11 +57,14 @@ export class Internet {
...
+   *
+   * @param firstName The optional first name to use. If not specified, a random one will be chosen.
+   * @param lastName The optional last name to use. If not specified, a random one will be chosen.
    *
-   * @method faker.internet.exampleEmail
-   * @param firstName
-   * @param lastName
+   * @example
+   * faker.internet.exampleEmail() // 'Helmer.Graham23@example.com'
+   * faker.internet.exampleEmail('Jeanne', 'Doe') // 'Jeanne96@example.net'
```
Comment (prisis) [inline src/internet.ts:67]: Same on this example

File: src/internet.ts (hunk 1 above)
Comment (ST-DDT) [reply src/internet.ts:41]: This is based on userName(). The name might get modified or partially ommited.
https://github.com/faker-js/faker/blob/ecf0b691ebeeca269472507745433e449508b0b0/src/internet.ts#L91-L93

File: src/internet.ts (hunk 2 above)
Comment (ST-DDT) [reply src/internet.ts:67]: Same here.

**Review by prisis — APPROVED** [review]

**Pull request merged**